Rules/CommonScripts/HelpfulDeathTips.cfg

```diff
@@ -98,5 +97,20 @@ tips = To hide these tips, and other in-game help, hit F1.;
 		Knights can use their swords to destroy wooden blocks and doors, but not stone;
 		Doors and siege weapons are vulnerable to explosives.;
 		Bison can be tamed with burgers.;
-		Resources can be resupplied by changing classes in workshops, not just at the tent!;
+		You can get resupplies in class workshops, not just at the tent!;
 		You can highlight useful items for your class by holding the pickup key.;
+		As a knight, you can always jab the enemy after successfully slashing them to deal extra heart of damage.;
+		You should always have one transport tunnel near your tent and another one at your tower for quick transportation.;
+		It's good to have some protection against enemies on your transport tunnels, such as traps or mines.;
+		You can throw saws at logs to quickly turn them into usable wood.;
+		Knights can cut down trees quicker than builders. If your team has a tree farm, cut down trees as knight to save time for your builders.;
+		Double slashing deals damage faster than normal slashing or jabbing.;
+		A drill will overheat very quickly when drilling doors and platforms.;
+		You can use a water bucket to extinguish lit kegs.;
+		Carrying a sponge in your inventory greatly reduces the length of next 2 stuns from water bombs or water arrows.;
+		Shielding water bombs and water arrows halves the stun time.;
+		Stone doors are durable, but very expensive - dont place too many of them in one doorway.;
+		Placing a mine on your flag will make it difficult for the enemy to take it without dying.;
+		As an archer, you can shoot enemy mines to blow them up. But be careful, as mine's explosion can still kill you and your teammates!;
```

Comment:
"But be careful, as ~~mine's~~ **the** explosion can still kill you and your teammates!"